src/OSPSuite.SimModelNative/src/DESolver.cpp

```diff
@@ -532,8 +541,8 @@ namespace SimModelNative
 			ydot[i] = 0.;
 
 		//set value of sensitivity parameters
-		for (i = 0; i < _sensitivityParameters.size(); i++)
-			_sensitivityParameters[i]->SetInitialValue(p[i]);
+		//for (i = 0; i < _sensitivityParameters.size(); i++)
+		//	_sensitivityParameters[i]->SetInitialValue(p[i]);
```

Comment:
Why did you remove setting parameters values?

Reply:
Temporal change. The algorithm is setting the values of the parameters outside the limits. I am reading to check how to tell CVODEs not to go off the limits. Otherwise we need to handle internally but ideally, we should restrict the search.

Reply:
You mean, parameters become e.g. <0 etc.? Interesting. Do you have an example?
If you don't find a solution you might ask the developers under https://github.com/LLNL/sundials/issues
But in any case this would require interface extension of SimModel, because as for now SimModel does not have any knowledge about allowed parameter bounds.